src/ore/src/tracing.rs

```diff
@@ -427,59 +473,77 @@ where
         None
     };
 
-    let (sentry_guard, sentry_layer) = if let Some(sentry_config) = config.sentry {
-        let guard = sentry::init((
-            sentry_config.dsn,
-            sentry::ClientOptions {
-                attach_stacktrace: true,
-                release: Some(config.build_version.into()),
-                environment: sentry_config.environment.map(Into::into),
-                integrations: vec![Arc::new(DebugImagesIntegration::new())],
-                ..Default::default()
-            },
-        ));
-
-        sentry::configure_scope(|scope| {
-            scope.set_tag("service_name", config.service_name);
-            scope.set_tag("build_sha", config.build_sha.to_string());
-            scope.set_tag("build_time", config.build_time.to_string());
-            for (k, v) in sentry_config.tags {
-                scope.set_tag(&k, v);
-            }
-        });
+    let (sentry_guard, sentry_layer, sentry_reloader): (_, _, DirectiveReloader) =
+        if let Some(sentry_config) = config.sentry {
+            let guard = sentry::init((
+                sentry_config.dsn,
+                sentry::ClientOptions {
+                    attach_stacktrace: true,
+                    release: Some(config.build_version.into()),
+                    environment: sentry_config.environment.map(Into::into),
+                    integrations: vec![Arc::new(DebugImagesIntegration::new())],
+                    ..Default::default()
+                },
+            ));
+
+            sentry::configure_scope(|scope| {
+                scope.set_tag("service_name", config.service_name);
+                scope.set_tag("build_sha", config.build_sha.to_string());
+                scope.set_tag("build_time", config.build_time.to_string());
+                for (k, v) in sentry_config.tags {
+                    scope.set_tag(&k, v);
+                }
+            });
```

Comment:
(not covered by unit tests but I am aware that this code was not newly introduced)